apps/studio/components/interfaces/DataWarehouse/TestCollectionDialog.tsx

```diff
@@ -94,83 +98,99 @@ export function TestCollectionDialog({
+                {accessTokens?.map((token: any) => (
+                  <SelectItem key={token.id + '-token'} value={token.token}>
+                    {token.description || 'No description'}
+                  </SelectItem>
+                ))}
+              </SelectContent>
+            </Select>
+          </FormItemLayout>
 
-          <FormItemLayout label="Ingest URL" isReactForm={false}>
-            <Input
+          <FormItemLayout layout="horizontal" label="Ingest API URL" isReactForm={false}>
+            <Input_Shadcn_
+              size="tiny"
               className="font-mono tracking-tighter"
               value={BASE_WAREHOUSE_URL}
               readOnly
-              copy
             />
           </FormItemLayout>
-
-          <div className="flex gap-2 items-center">
-            <Checkbox_Shadcn_
-              name="showAccessToken"
-              id="showAccessToken"
-              checked={showAccessToken}
-              onCheckedChange={() => setShowAccessToken(!showAccessToken)}
+          <FormItemLayout layout="horizontal" label="Collection ID" isReactForm={false}>
+            <Input_Shadcn_
+              size="tiny"
+              className="font-mono tracking-tighter"
+              value={selectedCollection}
+              readOnly
             />
-            <Label_Shadcn_ htmlFor="showAccessToken">Show access token</Label_Shadcn_>
+          </FormItemLayout>
+        </SheetSection>
+
+        <Separator />
+
+        <SheetSection>
+          <div className="flex justify-between items-center">
+            <div className="flex gap-2 items-center">
+              <Checkbox_Shadcn_
+                name="showAccessToken"
+                id="showAccessToken"
+                checked={showAccessToken}
+                onCheckedChange={() => setShowAccessToken(!showAccessToken)}
+              />
+              <Label_Shadcn_ htmlFor="showAccessToken">Show access token</Label_Shadcn_>
+            </div>
+            <CopyButton type="default" text={getcURL(testAccessToken)} />
           </div>
 
-          <div className="relative">
+          <div className="mt-4 relative">
             <CodeBlock
               hideCopy
-              className={'p-1 language-bash prose transition-colors'}
+              className={'p-1 language-bash prose max-w-full transition-colors'}
               language="bash"
             >
               {getcURL(showAccessToken ? testAccessToken : '********************')}
             </CodeBlock>
-            <CopyButton
-              type="default"
-              text={getcURL(testAccessToken)}
-              className="absolute top-2 right-2"
-            />
           </div>
-        </DialogSection>
-      </DialogContent>
-    </Dialog>
+        </SheetSection>
+        <SheetFooter className="mt-auto">
+          <p className="text-sm text-foreground-light mr-auto">
+            Once you send an event, refresh the page to see it in the table.
```

Comment:
wondering if it should be collection instead of table? keep the language consistent
```suggestion
            Once you send an event, refresh the page to see it in the collection.
```